client-service/src/main/java/com/telecominfraproject/wlan/client/validator/ClientSessionValidator.java

```diff
@@ -0,0 +1,27 @@
+package com.telecominfraproject.wlan.client.validator;
+
+import org.springframework.context.annotation.Configuration;
+
+import com.telecominfraproject.wlan.client.session.models.ClientSession;
+
+@Configuration
+public class ClientSessionValidator 
+{
+	protected static final int MAX_SSID_LENGTH = 32;
+	
+	public void validateClientSession(ClientSession clientSession) throws ClientSessionValidatorException
+	{
+		if (clientSession.getDetails().getSsid() != null)
+		{
```

Comment:
Consider adding NP checks for all dereferences in this method and checkSsidLength

Reply:
And unit tests as well. I know you mocked out the responses for these get calls, but it covers NPE if they ever occur.

Reply:
I think that mandatory fields should be checked in the @RequestBody like this https://stackoverflow.com/questions/64517537/springboot-validate-requestbody .

I will add some null checks & thrown exceptions since we don't do that